readthedocs/projects/models.py

```diff
@@ -624,6 +624,10 @@ def proxied_api_url(self):
         """
         return self.proxied_api_host.strip('/') + '/'
 
+    @property
+    def proxied_static_path(self):
```

Comment:
Could use a docstring explaining what this is for:

```suggestion
    def proxied_static_path(self):
    """Path for static files hosted on the user's doc domain."""
```